lib/eventbrite/api/model/base.rb

```diff
@@ -32,6 +32,10 @@ def update(id, object, opts={})
           @client.connection.post(url(opts, id), {:headers => @client.headers, :body => object.to_json})
         end
 
+        def del(opts={}, id)
```

Comment:
@KnightNoder 

Is there a specific reason why you named it `del` instead of `delete`

Reply:
@bvsatyaram 
While deleting event , 'stack level too deep'  was an error i was encountering until i removed the method name from delete to something else,still not exactly sure why it occured.